src/booking/Offers/OfferTypes.ts

```diff
@@ -354,6 +369,12 @@ export interface OfferPassenger {
    * Optionally providing one has been deprecated.
    */
   id: string
+
+  /**
+   * The fare type of the passenger
+   * Example: "contract_bulk"
+   */
+  fare_type: CreateOfferRequestPassengerFareType | null
```

Comment:
🎨 Would be nice to rename to `PassengerFareType` or something, but it's a nitpick.

Reply:
I kept that pattern to follow convention on how we wrote these types. If it was used somewhere else, I'd agree, but keeping it to the convention for now.